data/json/mutations.json

```diff
@@ -810,11 +810,20 @@
         "type" : "mutation",
         "id" : "SCHIZOPHRENIC",
         "name" : "Schizophrenic",
-        "points" : -5,
+        "points" : -3,
         "description" : "You will periodically suffer from delusions, ranging from minor effects to full visual hallucinations.  Some of these effects may be controlled through the use of Thorazine.",
         "starting_trait" : true,
         "valid" : false,
         "category" : ["MEDICAL"]
+    },{
+        "type" : "mutation",
+        "id" : "NARCOLEPTIC",
+        "name" : "Narcoleptic",
+        "points" : -2,
+        "description" : "You can randomly fall asleep without any reason.",
```

Comment:
I'd remove the "can", since the player doesn't really have a choice in the matter.

`You randomly fall asleep without any reason.`